zookeeper-docs/src/documentation/content/xdocs/zookeeperObservers.xml

```diff
@@ -111,7 +111,41 @@
       the ZooKeeper service.
     </para>
   </section>
-  
+  <section id="ch_ObserverMasters">
```

Comment:
general comments about documentation - great work! But we need put the document in markdown source now with ZOOKEEPER-3153 / 3154 is done.

Reply:
Will address in a rebase that fixes the conflicts - thanks for raising to me!